packages/desktop/src/renderer/components/CreateJoinCommunity/JoinCommunity/JoinCommunity.tsx

```diff
@@ -17,6 +17,8 @@ const JoinCommunity = () => {
   const joinCommunityModal = useModal(ModalName.joinCommunityModal)
   const createCommunityModal = useModal(ModalName.createCommunityModal)
 
+  const [showPassword, setShowPassword] = useState<boolean>(false)
```

Comment:
I don't really like naming it 'password' as we don't have any passwords here. It should be more like 'revealInputValue' or smthg

Reply:
When you name anything, always ask yourself a question "Will someone know what's going on here in three months?". Keeping variable and function names simple and literal is sometimes crucial